packages/console/src/pages/EnterpriseSso/Guide/XmlFileUploader/index.tsx

```diff
@@ -0,0 +1,73 @@
+import classNames from 'classnames';
+import { useRef, type ChangeEvent } from 'react';
+
+import Delete from '@/assets/icons/delete.svg';
+import FileIcon from '@/assets/icons/file-icon.svg';
+import UploaderIcon from '@/assets/icons/upload.svg';
+import Button from '@/ds-components/Button';
+
+import * as styles from './index.module.scss';
+
+type Props = {
+  onChange: (file?: File) => void;
```

Comment:
I think we should parse the file to string inside this component, no need to expose "File".